sample-data/fake.js

```diff
@@ -97,24 +137,39 @@ function write() {
         house.lotSize = Math.floor(Math.random() * (10000 - house.floorSize) + house.floorSize);
         house.daysListed = Math.floor(Math.random() * 364 + 1);
         house.saves = Math.floor(Math.random() * 150);
-        
+
         sampleData.push(house);
-        
+
         ok = writeStream.write(csvStringifier.stringifyRecords(sampleData), 'utf8');
-        
+
         sampleData = [];
-        
+
         i--;
-        
-        if(i % 1e4 === 0){
-            console.log(i);
-        } else if (i % 1e5 === 0) {
-            console.clear();
+
+        if (i % 1e5 === 0 || i < 5) {
+          console.log(i);
+          if (i === 0) {
+            resolve('Done');
+          }
         }
-    } while (i > 0 && ok);
+      } while (i > 0 && ok);
 
-    if (i > 0) {
+      if (i > 0) {
         writeStream.once('drain', write);
+      }
     }
-    console.log(Date.now());
+  }).then(resolve => {
+    let endTime = new Date();
+    let elapsedTime = new Date(endTime - startTime);
+    console.log(getDateTime(elapsedTime, recordQuantity));
+  });
+};
+
+async function run(outputDataPath, recordQuantity) {
+  console.log(outputDataPath);
+  await fake(outputDataPath, recordQuantity);
 }
+
+run(outputDataPath, recordQuantity);
+
+//module.exports = run;
```

Comment:
If you also export the run function, then you can use it for testing without diving into the command line from a script!